modules/samples/connectedcup/component/controller/src/main/java/org/coffeeking/controller/service/util/ConnectedCupServiceUtils.java

```diff
@@ -66,7 +67,7 @@ public static String sendCommandViaHTTP(final String deviceHTTPEndpoint, String
             } catch (ProtocolException e) {
                 String errorMsg =
                         "Protocol specific error occurred when trying to set method to GET" +
-                                " for:" + urlString;
+                        " for:" + urlString;
                 log.error(errorMsg);
                 throw new DeviceManagementException(errorMsg, e);
```

Comment:
@menakaj Is there a reason to log the error and throw the error as well?

Reply:
This code has been fixed in latest releases.